jenkins/bootstrap.py

```diff
@@ -330,10 +330,23 @@ def cat(self, path, generation):
         return self.call(cmd, output=True)
 
 
-    def upload_artifacts(self, path, artifacts):
+    def upload_artifacts(self, gsutil, path, artifacts):
         """Upload artifacts to the specified path."""
         # Upload artifacts
         if os.path.isdir(artifacts):
```

Comment:
if not os.path.isdir(artifacts): 
  return

(then shrink the indent level for everything else